src/cli/commands/deploy/deploy.ts

```diff
@@ -34,6 +34,11 @@ export async function deploy(options: SWACLIConfig) {
   // make sure appLocation is set
   appLocation = path.resolve(appLocation || process.cwd());
 
+  // make sure dataApiLocation is set
```

Comment:
I think we also use some default folder name for data-api-location. Do we also need to pass that folder name in default scenarios.

Reply:
By convention, if the folder `swa-db-connections` is in the root directory, it gets deployed (no need to pass default args to the StaticSitesClient)